spec/models/claim_spec.rb

```diff
@@ -165,8 +165,8 @@
 
     # Tests a single attribute, possibly should test multiple attributes
```

Comment:
That's an interesting test 🤔 - anyway, something for another day

Reply:
Yeah I just updated the spec so this functionality is still tested by something. Once everything has been moved over to form objects we should be able to lose this spec